test/functional/content_view/version/export_test.rb

```diff
@@ -0,0 +1,48 @@
+require File.join(File.dirname(__FILE__), '../../test_helper')
```

Comment:
There is `__dir__` that's equal to `File.dirname(__FILE__)`. Otherwise there's `require_relative` as well.

Reply:
That a good idea, I just took this from other test files, so gonna err on the side of consistency.